### DISCUSSION TITLE
Disable update button for non-downloadable beatmaps

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#23813] [Ideas]
Players may be confused by this
![image](https://github.com/ppy/osu/assets/74862110/f11a1176-d1cc-4912-adba-d8bcdc41d2b3)

**Comment by peppy** [comment]
Can you please link the beatmap and explain why it is not downloadable?

**Comment by Jan-Arbus** [comment]
Beatmap link: https://osu.ppy.sh/beatmapsets/25403#osu/86044
Reason for not downloading given by beatmaps: https://gist.githubusercontent.com/peppy/99e6959772083cdfde8a/raw
I guess something in this beatmaps violates copyright

**Reply by peppy** [reply]
Thanks.

Might be a bit hard to implement this (and it's a very edge case) but we'll keep it in mind.
